src/js/components/Layout/Footer.jsx

```diff
@@ -59,7 +60,7 @@ const Footer = ({
               <a
                 className={`${locale === language.code ? 'selected' : ''}`}
                 key={language.code}
-                href={`/openboxes/user/disableLocalizationMode?locale=${language.code}`}
+                href={`${DISABLE_LOCALIZATION}?locale=${language.code}`}
```

Comment:
Maybe we can move this `locale=${language.code}` to `DISABLE_LOCALIZATION`? I mean adding sth like:
```
export const DISABLE_LOCALIZATION = (languageCode) => `${CONTEXT_PATH}/user/disableLocalizationMode${languageCode ? `?locale=${languageCode}` : ''}`;
```
What do you think?
Or if there are more than one param, we can create a util for mapping those params